web-ng/src/app/shared/models/project.model.ts

```diff
@@ -29,11 +29,14 @@ export class Project {
 
   getForm(layerId: string, formId: string): Form {
     const layer = this.layers.get(layerId);
+    let form;
     if (!layer) {
       throw Error('Layer not found.');
     }
-    const form = layer.forms.get(formId);
-    if (!form) {
+    if (layer.forms) {
+      form = layer.forms.get(formId);
+    }
+    if (!layer.forms || !form) {
```

Comment:
We've already checked `layer.forms` above, so let's only check the output `form` here.